components/Lampe.js

```diff
@@ -0,0 +1,37 @@
+export default ({
+  rgba_value,
+  ...rest
+}) => (
+  <div>
+  <div id="ledrahmen">
+  <div id="led">
+  </div>
+  </div>
+  <style jsx>{`
+    #ledrahmen {
+      background-color: black;
+      position: absolute;
+      width: 200px;
+      height: 200px;
+      top: 100px;
+      left: 25px;
+      border: 1px solid black;
+      border-radius: 125px
+    }
+
+    #led {
+      background-color: rgba(${rgba_value}); /* rgb werte ohne transperenz, hierim kommentar mit transperenz rgba(230,140,140,1.0); */
```

Comment:
Rot, Gelb, Blau und tramsparenz sollen als einzelne Parameter an den Komponenten übergeben werden. Dynamische Werte bitte nicht in die Klasse (nicht ins css) sondern über das style attribut übergeben. Bei Fragen einfach melden.